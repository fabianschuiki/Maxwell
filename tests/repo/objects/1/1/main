{"ident":{"name":"helloWorld"},"inner":{"class":"IdentifierExpr","name":"innerIdent","someText":"blabla"}}