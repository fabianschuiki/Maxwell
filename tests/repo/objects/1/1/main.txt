Array
(
    [name] => someFunc
    [ident] => Array
        (
            [name] => helloWorld
            [class] => IdentifierExpr
            [id] => 1.1.ident
        )

)
