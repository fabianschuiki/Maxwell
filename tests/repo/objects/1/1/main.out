{"inner":{"name":"innerIdent","someText":"hahaha","class":"IdentifierExpr"},"ident":{"name":"helloWorld"}}